src/coprocessor/endpoint.rs

```diff
@@ -293,6 +309,7 @@ pub struct RequestTask {
     wait_time: Option<f64>,
     timer: Instant,
     statistics: Statistics,
+    scan_counter: ScanCounter,
```

Comment:
Why not use CopStats directly here?

Reply:
So we need to consider adjust `collect_statistics_into` and `collect_scan_count_into` into one method. Maybe do it in another PR is better?